d2l-test-reporting.config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "tool": "n/a",
+  "experience": "n/a",
```

Comment:
I couldn't find anything in the product taxonomy that made sense for these here. Any ideas @sarahboyd?

Reply:
That's a pretty common problem for us haha.

Reply:
I'd think UI Platform & Components for tool, under Software Services  - would that make sense?
![image](https://github.com/user-attachments/assets/d1692d76-2b66-4601-88ca-3db3b0cb0194)

Reply:
Ah yes. Didn't see that. That diagram makes me go blind. I'm glad your sight isn't lost on it

Reply:
I've just spent too much time looking through it 😆 🙃 🙈